R/checks.R

```diff
@@ -372,37 +364,37 @@ check_metrics <- function(x, object) {
   is_surv_metric_set <- inherits(x, c("survival_metric_set"))
 
   if (!is_numeric_metric_set && !is_class_prob_metric_set && !is_surv_metric_set) {
-    rlang::abort("The `metrics` argument should be the results of [yardstick::metric_set()].")
+    cli::cli_abort("The {.arg metrics} argument should be the results of
+                   {.fn yardstick::metric_set}.")
```

Comment:
Previous messages has `[pkg:fun()]` format. Do we want to move to that? I don't think that they were clickable.